internal/core/converter/tensorflow/tensorflow.go

```diff
@@ -56,20 +63,17 @@ const (
 	ThreeDim
 )
 
+var loadFunc = func(exportDir string, tags []string, options *SessionOptions) (*tf.SavedModel, error) {
```

Comment:
consider to create an interface for the loading function.
@hlts2 what do you think?